suite-native/device-manager/src/components/WalletItem.tsx

```diff
@@ -0,0 +1,70 @@
+import { Pressable } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { HStack, Radio, Text } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Icon } from '@suite-common/icons';
+import { AcquiredDevice } from '@suite-common/suite-types';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { selectDevice } from '@suite-common/wallet-core';
+
+const walletItemStyle = prepareNativeStyle<{ isSelected: boolean }>((utils, { isSelected }) => ({
+    paddingHorizontal: utils.spacings.medium,
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: 60,
+    gap: 12,
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    borderWidth: 1,
+    borderRadius: 12,
+    borderColor: utils.colors.borderElevation1,
+    extend: {
+        condition: isSelected,
+        style: { borderWidth: 2, borderColor: utils.colors.borderSecondary },
+    },
+}));
+
+type WalletItemProps = {
+    device: AcquiredDevice;
+    isSelectable?: boolean;
+    onPress?: () => void;
```

Comment:
Undefined is not handled but also shouldn't be needed, so please make this required

```suggestion
    onPress: () => void;
```

Reply:
5825a46c34